src/witan/plot/chart.clj

```diff
@@ -0,0 +1,96 @@
+(ns witan.plot.chart
+  (:require [cljplot.build :as plotb]
+            [cljplot.config :as cfg]
+            [cljplot.core :as plot]
+            [cljplot.render :as plotr]
+            [clojure.string :as s]
+            [witan.plot.colors :as colors]
+            [witan.plot.series :as ps]))
+
+(defn zero-index-numerical-y-axes [prepped-data]
+  (let [[t [_bottom top]] (get-in prepped-data [:extents :y 0])]
+    (if (= :numerical t)
+      (plotb/update-scale prepped-data :y :domain [0 (+ (* 0.025 top) top)])
+      prepped-data)))
+
+(defn update-chart-y-axis-ticks [chart-spec series]
+  (let [[t [_bottom top]] (get-in chart-spec [:extents :y 0])]
+    (if (and (= :numerical t) (< top 10))
+      (plotb/update-scale chart-spec :y :domain [0 10])
+      chart-spec)))
+
+(defn update-chart-x-axis-ticks [chart-spec series]
+  (let [x-count (->> series
+                     (mapcat (fn [x] (second x)))
+                     (map (fn [y] (first y)))
+                     (into #{})
+                     count)]
+    (if (< x-count 10)
+      (plotb/update-scale chart-spec :x :ticks x-count)
+      chart-spec)))
+
+(defn zero-y-index [{::ps/keys [series legend-spec]
+                     ::keys [x-axis
+                             y-axis
+                             size
+                             legend-label ;; new
+                             title]
+                     :as _chart-spec}]
+  (let [_config (swap! cfg/configuration
+                       (fn [c]
+                         (-> c
+                             (assoc-in [:legend :font] "Open Sans Bold")
+                             (assoc-in [:legend :font-size] 24))))
```

Comment:
How is this used?

Reply:
I feel like this is set up to overwrite any prior config one may have set up and if so is somewhat invisible

Reply:
This fixes the font on the legend. It isn't great atm (having it as an atom that we have to tweak isn't great either). We kept getting caught out by not having this in our configs though. I'll have a think about how it should be better done in the future.